docs/shared/make_solo_mosaic.yaml

```diff
@@ -0,0 +1,17 @@
+make_solo_mosaic:
+  config:
+    dir: /path/to/input
+    num_tiles: 1
+    mosaic_name: ocean_mosaic
+    periodx: 360
+    periody: 360
+    tile_file: C403_grid.tile7.halo6.nc
```

Comment:
These four flags are optional and the other two, `dir` and `num_tiles`, are required. 
1. Should the optional flags be included for in the yaml? 
2. If yes, should all 6 be sorted alphabetically, regardless of importance? 
3. Do the optional flags need to be accounted for in the schema and in the corresponding`yaml/components` file?

Reply:
I vote yes on all three. The schema can make the required parameters required, and can specify the value type for each/all of them.